src/main/resources/org/rust/ide/liveTemplates/output.xml

```diff
@@ -31,6 +31,24 @@
         </context>
     </template>
 
+    <template name="pdv" description="debug println! value" toReformat="true" toShortenFQNames="true"
+              value="println!(&quot;$EXPRESSION_COPY$ = {:?}&quot;, $EXPRESSION$);">
```

Comment:
So I've merged this templates, and I think it's kinda best of worlds: cursor is first placed on EXPRESSION, which is copied to EXPRESSION_COPY, but then it is placed over the copy to allow editing.  

https://github.com/intellij-rust/intellij-rust/commit/f85119a2469f0c9507c746887022388871651651